Flutter/flutter_gallery_app/lib/src/presentation/wrap/screen/wrap_page.dart

```diff
@@ -0,0 +1,303 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_gallery_app/src/presentation/container/widgets/view_option_swich.dart';
+import 'package:flutter_gallery_app/src/presentation/wrap/bloc/wrap_bloc.dart';
+import 'package:flutter_gallery_app/src/presentation/wrap/bloc/wrap_event.dart';
+import 'package:flutter_gallery_app/src/presentation/wrap/bloc/wrap_state.dart';
+
+@RoutePage()
+class WrapPage extends StatelessWidget {
+  const WrapPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider<WrapBloc>(
+      create: (context) => WrapBloc(),
+      child: const WrapView(),
+    );
+  }
+}
+
+class WrapView extends StatefulWidget {
```

Comment:
You should separate the 'WrapView' class into a separate file. And then change name "WrapView" to "WrapWidget"

Reply:
I have resolved the problem in new commit